dart/dynamics/AssimpInputResourceAdaptor.cpp

```diff
@@ -0,0 +1,207 @@
+#include <cassert>
+#include <iostream>
+#include <assimp/IOStream.hpp>
+#include "dart/common/Console.h"
+#include "AssimpInputResourceAdaptor.h"
+
+namespace dart {
+namespace dynamics {
+
+/*
+ * AssimpInputResourceRetrieverWrapper
+ */
+AssimpInputResourceRetrieverAdaptor::AssimpInputResourceRetrieverAdaptor(
+      const common::ResourceRetrieverPtr& _resourceRetriever)
+  : mResourceRetriever(_resourceRetriever)
+{
```

Comment:
I typically put a ``// Do nothing`` in places like this to make it clear that the programmer didn't simply forget to implement it.